spring-cloud-netflix-eureka-client/src/main/java/org/springframework/cloud/netflix/eureka/config/DiscoveryClientOptionalArgsConfiguration.java

```diff
@@ -69,11 +70,11 @@ public TlsProperties tlsProperties() {
 	@ConditionalOnProperty(prefix = "eureka.client", name = "webclient.enabled", matchIfMissing = true,
 			havingValue = "false")
 	public RestTemplateDiscoveryClientOptionalArgs restTemplateDiscoveryClientOptionalArgs(TlsProperties tlsProperties,
-			EurekaClientHttpRequestFactorySupplier eurekaClientHttpRequestFactorySupplier)
-			throws GeneralSecurityException, IOException {
+			EurekaClientHttpRequestFactorySupplier eurekaClientHttpRequestFactorySupplier,
```

Comment:
Please add your full name and surname with `@author` tag to the javadoc of all the files you've modified. Please update the year in the license comments of the files you've modified.

Reply:
@ZIRAKrezovic Actually, I'm just going to do it for you, since I'd like to make sure we include this in the upcoming release.